install/kubernetes/helm/istio/charts/pilot/values.yaml

```diff
@@ -10,6 +10,11 @@ autoscaleMax: 5
 image: pilot
 sidecar: true
 traceSampling: 100.0
+# specify the applications namespace list pilot manages, separated by comma, if not empty, 
+# the value must contain 'istio-system'; if empty, pilot watches all namespaces
```

Comment:
Why does it have to contain istio-system ? And keep in mind pilot can be installed in any namespace.

Reply:
The namespace where control plane located in must be contained, or some required endpoints can't be pushed to envoy, eg `istio-policy`, `istio-telemetry`...